<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sniffer_res.qrc">
    <normaloff>:/images/sniffer.png</normaloff>:/images/sniffer.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>142</x>
         <y>6</y>
         <width>123</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Baud</string>
       </property>
      </widget>
      <widget class="QComboBox" name="stopBits">
       <property name="geometry">
        <rect>
         <x>401</x>
         <y>60</y>
         <width>123</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="dataBits">
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>60</y>
         <width>124</width>
         <height>24</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>6</y>
         <width>123</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity</string>
       </property>
      </widget>
      <widget class="QComboBox" name="baud">
       <property name="geometry">
        <rect>
         <x>142</x>
         <y>60</y>
         <width>123</width>
         <height>24</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>271</x>
         <y>6</y>
         <width>124</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Data bits</string>
       </property>
      </widget>
      <widget class="QComboBox" name="serialPort">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>60</y>
         <width>123</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>401</x>
         <y>6</y>
         <width>123</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop bits</string>
       </property>
      </widget>
      <widget class="QComboBox" name="parity">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>60</y>
         <width>123</width>
         <height>24</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>odd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>even</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>6</y>
         <width>123</width>
         <height>48</height>
        </rect>
       </property>
       <property name="text">
        <string>Serial port</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>60</y>
         <width>83</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Active</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>8</number>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Raw data:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearRawData">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Clear Raw Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLockRaw">
            <property name="text">
             <string>lock</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Commands:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearCmdData">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Clear Commands Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkLockCmds">
            <property name="text">
             <string>lock</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="busMonTable">
          <column>
           <property name="text">
            <string>Slave ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Cmd</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Data</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="sniffer_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClearRawData</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
